fundamentals/fundamentals-bio/src/main/scala/izumi/functional/bio/impl/BIOAsyncMonix.scala

```diff
@@ -0,0 +1,132 @@
+package izumi.functional.bio.impl
+
+import java.util.concurrent.CompletionStage
+
+import izumi.functional.bio.BIOExit.MonixExit._
+import izumi.functional.bio.{BIOAsync, BIOExit, BIOFiber}
+import monix.bio
+import monix.bio.{Cause, IO, Task}
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.Try
+
+object BIOAsyncMonix extends BIOAsyncMonix
```

Comment:
I think this also needs to be exposed as an implicit (in `BIORootInstances`)

Reply:
Done